views/layouts/main.hbs

```diff
@@ -1,10 +0,0 @@
-<!DOCTYPE HTML>
-<html>
-  <head>
-    <meta charset="utf8" />
-    <title>Patient Login</title>
-  </head>
-  <body>
-    {{{body}}} 
```

Comment:
Don't delete, this is the main template for the page, we will eventually add the CSS file to this and any front-end javascript